templates/repo/diff/compare.tmpl

```diff
@@ -194,9 +194,9 @@
 				</div>
 				<div class="four wide column middle aligned text right">
 				{{- if .PullRequest.HasMerged -}}
-				<a href="{{$.RepoLink}}/pulls/{{.PullRequest.Issue.Index}}" class="ui button purple show-form">{{svg "octicon-git-merge" 16}} {{ctx.Locale.Tr "repo.pulls.view"}}</a>
+				<a href="{{$.RepoLink}}/pulls/{{.PullRequest.Issue.Index}}" class="ui button primary show-form">{{svg "octicon-git-merge" 16}} {{ctx.Locale.Tr "repo.pulls.view"}}</a>
```

Comment:
Sorry but it is not right.

"purple" means it has been merged.

Reply:
But does it really need to indicate on the button? The button will not merge it so I think it's wrong indication.

Reply:
Such UI is widely used. For example, the "buttons" are provided by "label". It does provide a button-like UI.

![image](https://github.com/go-gitea/gitea/assets/2114189/c0592fbf-b8ae-4015-b5b7-3328e9fcb3a9)

Reply:
Yes, but those are labels, not buttons. Nothing bad will happen if I click the red button on a merged PR, it just takes me to the PR.

Reply:
So I guess I could redesign these messages to show a open/merged/closed label instead of this misleadingly colored button.

Reply:
> Yes, but those are labels, not buttons, e.g. they don't have actions associated.

They have, the same as this `<a>` like here.

Reply:
> So I guess I could redesign these messages to show a open/merged/closed label instead of this misleadingly colored button.

I could agree. Using a proper label link seems good.